### DISCUSSION TITLE
CSS and JavaScript not working for my portfolio website :/

### DISCUSSION THREAD

**Discussion by YonathenD** [community/community#42315] [New to GitHub]
Hey guys, I'm new to how GitHub pages work. On my local machine my website shows up perfectly but when I try to make it show up on GitHub Pages, only the HTML is visible. Does anyone know of a solution? Below is my repo. 

[https://github.com/YonathenD/yonathend.github.io](url)

**Comment by airtower-luna** [comment]
As far as [the stylesheet is concerned](https://github.com/YonathenD/yonathend.github.io/blob/8b12e675b3575911eb65abcb7fb7ac3d5b184874/index.html#L7), the quotes for including it are broken:

```html
    <link rel=“stylesheet” type=“text/css” href=“style.css”>
```

You need to use plain quotes (`"`) for HTML attributes:

```html
    <link rel="stylesheet" type="text/css" href="style.css">
```

Oddly in the rest of your file the quotes seem to be fine, I wonder what happened to that line? :sweat_smile:
